src/main/java/no/fdk/referencedata/eu/AbstractEuHarvester.java

```diff
@@ -16,13 +16,13 @@
 @Component
 @Slf4j
 public abstract class AbstractEuHarvester<T> {
-    private static final String DOWNLOAD_API = "https://op.europa.eu/o/opportal-service/euvoc-download-handler";
+    private static final String SPARQL_API = "http://publications.europa.eu/webapi/rdf/sparql";
 
     public abstract String getVersion();
 
-    public Resource getSource(final String cellarURI, final String fileName) {
+    public Resource getSource(final String sparqlQuery) {
```

Comment:
Kanskje ferne attr og bruke abstract method for sparqlQuery?

Reply:
5951379